sycl/test-e2e/Complex/sycl_complex_literals_test.cpp

```diff
@@ -0,0 +1,24 @@
+// DEFINE: %{mathflags} = %if cl_options %{/clang:-fno-fast-math%} %else %{-fno-fast-math%}
+
+// RUN: %{build} -fsycl-device-code-split=per_kernel %{mathflags} -o %t.out
+// RUN: %{run} %t.out
```

Comment:
```suggestion
// RUN: %{build} -o %t.out
// RUN: %{run} %t.out
```

I don't quite understand why do we need special math flags in this particular test; nor is device code split needed here, because there are no kernels.

Moreover, I would suggest to move runtime checks to `static_assert` statements and refactor `RUN` lines into:
```
// RUN: %{build} -fsyntax-only
```

To simplify the test even further and make it quicker to pass.

It also should probably be moved to a regular `test/` folder, since it doesn't require any devices to be launched.

Reply:
Thanks for pointing to `test/` directory. Test is now moved there, with compile-time checks via `static_assert`.